queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
**Changes Required**
- [ ] This query will return all records containing `mon` yet the requirement is to return only records ending with `mon`. Let us please try and fix this.